src/Sylius/Behat/Context/Transform/TaxCategoryContext.php

```diff
@@ -35,7 +35,7 @@ public function __construct(TaxCategoryRepositoryInterface $taxCategoryRepositor
     /**
      * @Transform /^"([^"]+)" tax category$/
      * @Transform /^tax category "([^"]+)"$/
-     * @Transform :taxCategory tax category
+     * @Transform :taxCategory
```

Comment:
This way we won't be able to retrieve just a name without transforming it eg. in the steps that creates it.